release-tools/prow.sh

```diff
@@ -1077,106 +852,49 @@ main () {
-                info "Version ${CSI_PROW_KUBERNETES_VERSION}, installing CRDs and snapshot controller"
-                install_snapshot_crds
-                install_snapshot_controller
+            # Install necessary CRDs and controllers
+            # For Kubernetes 1.19+, we will install the CRDs and controller.
+            if version_gt "${COSI_PROW_KUBERNETES_VERSION}" "1.16.255" || "${COSI_PROW_KUBERNETES_VERSION}" == "latest"; then
+                echo "Srini Install CRDs and Controller"
+                info "Version ${COSI_PROW_KUBERNETES_VERSION}, installing CRDs and cosi controller"
+                install_crds
+                install_controller
             else
-                info "Version ${CSI_PROW_KUBERNETES_VERSION}, skipping CRDs and snapshot controller"
+                info "Version ${COSI_PROW_KUBERNETES_VERSION}, skipping CRDs and cosi controller"
             fi
 
-            # Installing the driver might be disabled.
-            if ${CSI_PROW_DRIVER_INSTALL} "$images"; then
-                collect_cluster_info
-
-                if sanity_enabled; then
-                    if ! run_sanity; then
-                        ret=1
-                    fi
-                fi
-
-                if tests_enabled "parallel"; then
-                    # Ignore: Double quote to prevent globbing and word splitting.
-                    # shellcheck disable=SC2086
-                    if ! run_e2e parallel ${CSI_PROW_GINKO_PARALLEL} \
-                         -focus="External.Storage" \
-                         -skip="$(regex_join "${CSI_PROW_E2E_SERIAL}" "${CSI_PROW_E2E_ALPHA}" "${CSI_PROW_E2E_SKIP}")"; then
-                        warn "E2E parallel failed"
-                        ret=1
-                    fi
-
-                    # Run tests that are feature tagged, but non-alpha
-                    # Ignore: Double quote to prevent globbing and word splitting.
-                    # shellcheck disable=SC2086
-                    if ! run_e2e parallel-features ${CSI_PROW_GINKO_PARALLEL} \
-                         -focus="External.Storage.*($(regex_join "${CSI_PROW_E2E_FOCUS}"))" \
-                         -skip="$(regex_join "${CSI_PROW_E2E_SERIAL}")"; then
-                        warn "E2E parallel features failed"
-                        ret=1
-                    fi
-                fi
-
-                if tests_enabled "serial"; then
-                    if ! run_e2e serial \
-                         -focus="External.Storage.*($(regex_join "${CSI_PROW_E2E_SERIAL}"))" \
-                         -skip="$(regex_join "${CSI_PROW_E2E_ALPHA}" "${CSI_PROW_E2E_SKIP}")"; then
-                        warn "E2E serial failed"
-                        ret=1
-                    fi
+            echo "Srini Collect Cluster Info"
+            collect_cluster_info 
+            echo "Srini Done Collect Cluster Info"
```

Comment:
Looks like these are your temporary debug messages.
Can you mark this PR as WIP?  Looks like you are still testing.

Reply:
Fixed.